retry

```diff
@@ -6,24 +6,28 @@ options:
     -h,--help               Show this help
     -r,--retries=RETRIES    How many retries to do on command failure after the
                             initial try. Defaults to 3.
-    -s,--sleep=SLEEP        How many seconds to sleep between retries
+    -s,--sleep=SLEEP        How many seconds to sleep between retries. Defaults
+                            to 3 seconds.
+    -e,--exponential        Enable simple binary exponential back-off
+                            algorithm. Disabled by default.
     "
     exit "$1"
 }
 
 # parse arguments
 opts=$(getopt \
-    --options h,r:,s: \
-    --longoptions help,retries:,sleep: \
+    --options h,r:,s:,e \
+    --longoptions help,retries:,sleep:,exponential \
     --name "$(basename "$0")" -- "$@") || usage 1
 eval set -- "$opts"
 while [ $# -gt 0 ]; do
   case "$1" in
-    -h | --help    ) usage 0; shift   ;;
-    -r | --retries ) retries=$2; shift 2 ;;
-    -s | --sleep   ) sleep=$2; shift 2 ;;
-    --             ) shift; break ;;
-    *              ) break ;;
+    -h | --help        ) usage 0; shift ;;
+    -r | --retries     ) retries=$2; shift 2 ;;
+    -s | --sleep       ) sleep=$2; shift 2 ;;
+    -e | --exponential ) exponential=2; shift ;;
```

Comment:
… aligned with what I suggested above then this would just read out `$2`

Reply:
Yes, I was thinking if I should make the base configurable and decided for simplicity.. As you requested more features, I will extend it.